src/Platform/AcquiaCloudMultiSitePlatform.php

```diff
@@ -50,25 +50,37 @@ public function execute(Command $command, InputInterface $input, OutputInterface
     $sshUrl = $environment->sshUrl;
     [, $url] = explode('@', $sshUrl);
     [$application] = explode('.', $url);
+
     $sites = $this->getPlatformMultiSites($environment, $application, $output, $vendor_path[$env_id]);
     if (!$sites) {
       $output->writeln('<warning>No sites available. Exiting...</warning>');
       return 1;
     }
 
+    $group_name = $input->getOption('group');
     $input_uri = $input->getOption('uri');
+
+    if (!$input_uri && $group_name) {
+      $alias = $this->getAlias();
+      $platform_id = self::getPlatformId();
+      $sites = $this->filterSitesByGroup($group_name, $sites, $output, $alias, $platform_id);
+      if (empty($sites)) {
+        return 3;
```

Comment:
Please document return codes in doc comment.

Reply:
On the long term it would be even better, if we stored these return codes in constants with appropriate names.